py/DREAM/Output/OtherQuantities.py

```diff
@@ -112,7 +112,7 @@ def setQuantity(self, name, data, attributes=None, datatype=None):
             elif data.ndim == 4 and self.momentumgrid is not None:
                 o = self.SPECIAL_TREATMENT[name](name=name, data=data, description=desc, grid=self.grid, output=self.output, momentumgrid=self.momentumgrid)
             else:
-                o = self.SPECIAL_TREATMENT[name](name=name, data=data, description=desc, grid=self.grid, output=self.output)
+                o = self.SPECIAL_TREATMENT[name](name=name, data=data, description=desc, grid=self.grid, output=self.output, momentumgrid=self.momentumgrid)
```

Comment:
This assumes that all classes in ``SPECIAL_TREATMENT`` accept the ``momentumgrid`` keyword argument, which is (I believe) why it was added to the ``AvalancheGrowthRate`` and ``LCFSLoss`` constructors in this PR.

If this is the case, then it should also be added in the ``OtherQuantity`` constructor as well. Otherwise, I suspect that there will be a TypeError raised whenever ``f_hot_ripple_pmn`` or ``f_re_ripple_pmn`` is included.

Reply:
Thanks! You're, absolutely right, I will fix this ;)